src/javascript/app/Stores/Modules/SmartChart/smart-chart-store.js

```diff
@@ -334,6 +336,12 @@ export default class SmartChartStore extends BaseStore {
                 time    : ServerTime.get().unix(),
             }));
         }
+        if (request_object.active_symbols) {
+            if (this.should_refresh_active_symbols) {
+                return WS.sendRequest(request_object, { forced: true });
+            }
+            return BinarySocket.wait('active_symbols').then(active_symbols => active_symbols);
```

Comment:
should this return a promise (just `BinarySocket.wait`) instead of the active_symbols response?

Reply:
yeah nice catch, thanks!